libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -23,8 +23,11 @@ PbftSyncPacketHandler::PbftSyncPacketHandler(
       network_sync_level_size_(network_sync_level_size) {}
 
 void PbftSyncPacketHandler::validatePacketRlpFormat(const PacketData &packet_data) const {
-  if (constexpr size_t required_size = 3; packet_data.rlp_.itemCount() != required_size) {
-    throw InvalidRlpItemsCountException(packet_data.type_str_, packet_data.rlp_.itemCount(), required_size);
+  constexpr size_t standard_required_size = 3;
+  constexpr size_t last_block_required_size = 4;
+  if (packet_data.rlp_.itemCount() != standard_required_size &&
+      packet_data.rlp_.itemCount() != last_block_required_size) {
+    throw InvalidRlpItemsCountException(packet_data.type_str_, packet_data.rlp_.itemCount(), standard_required_size);
```

Comment:
The if condition checks itemCount() != 3 && itemCount() != 4, but the throw exception will only show standard_required_size

Reply:
that is ok